js/components/main.js

```diff
@@ -374,6 +376,11 @@ class Main extends ImmutableComponent {
       windowActions.setContextMenuDetail()
     })
 
+    ipc.on(messages.IMPORTER_LIST, (e, detail) => {
```

Comment:
this would also be better as an appAction. We want to avoid adding more ipc handlers